rubin_scheduler/scheduler/example/example_scheduler.py

```diff
@@ -104,46 +104,52 @@ def standard_bf(
 
     Parameters
     ----------
-    nside : int (DEFAULT_NSIDE)
-        The HEALpix nside to use
-    nexp : int (1)
-        The number of exposures to use in a visit.
-    exptime : float (30.)
-        The exposure time to use per visit (seconds)
-    filtername : list of str
-        The filternames for the first set
-    filtername2 : list of str
-        The filter names for the second in the pair (None if unpaired)
-    n_obs_template : dict (None)
-        The number of observations to take every season in each filter
-    season : float (300)
-        The length of season (i.e., how long before templates expire) (days)
-    season_start_hour : float (-4.)
-        For weighting how strongly a template image needs to be
-        observed (hours)
-    sesason_end_hour : float (2.)
-        For weighting how strongly a template image needs to be
-        observed (hours)
-    moon_distance : float (30.)
-        The mask radius to apply around the moon (degrees)
-    m5_weight : float (3.)
-        The weight for the 5-sigma depth difference basis function
-    footprint_weight : float (0.3)
+    nside : `int`
+        The HEALpix nside to use. Defaults to DEFAULT_NSIDE
+    filtername : `str`
+        The filter name for the first observation. Default "g".
```

Comment:
Where we have a default value, it will show up in the docstring when this is built in the docs (see first screenshot). Because the docstring and the call are right next to each other, we don't have to repeat the default values in the docstring (and, as you can see in the second screenshot, it can make it all look messier). 

I think if we're using None so that it defaults to some other value within the code, that "Default None will be set to xxxx" is useful, or  it can be useful to know if we're setting the default for a good reason that other people should know (and then probably not change) -  but otherwise we should probably get out of the habit of repeating our defaults. 

<img width="845" alt="Screenshot 2024-10-21 at 7 06 12 PM" src="https://github.com/user-attachments/assets/720badcf-8527-4c86-b3b2-7e6d58abc713">
 
<img width="855" alt="Screenshot 2024-10-21 at 7 08 24 PM" src="https://github.com/user-attachments/assets/d2c3537b-8ad4-4584-9134-286f7dcddb59">

Reply:
But if you pull up the docstring from the command line or in a notebook the call signature and the relevant section of the docstring can get totally separated. 

![image](https://github.com/user-attachments/assets/71fd09d3-4043-4e03-87d3-d10d9d2206a7)

Reply:
I guess I was thinking that 
<img width="1783" alt="Screenshot 2024-10-22 at 2 21 22 PM" src="https://github.com/user-attachments/assets/81159eb1-1130-460f-af1d-64aec222d7c2">
is still less readable than this (where removing the additional Default lines means that the whole thing is shorter)
<img width="1782" alt="Screenshot 2024-10-22 at 2 24 24 PM" src="https://github.com/user-attachments/assets/6d686f91-4f4c-45d0-8a69-fe44800506e6">

My main problem is that the defaults that are in these sections tend to get outdated. If these are kept up to date, I'll be ok.